docs/guide/api-environment-runtimes.md

```diff
@@ -182,8 +182,9 @@ type ModuleRunnerTransport = unknown
 interface ModuleRunnerOptions {
   /**
    * Root of the project
+   * @deprecated not used anymore and to be removed
    */
-  root: string
+  root?: string
```

Comment:
Since `root` is deprecated and does not need to be set anymore, I think we can remove `root` from all the examples and types in the docs.